src/libs/actions/IOU.js

```diff
@@ -201,14 +202,15 @@ function payIOUReport({
             if (response.jsonCode !== 200) {
                 throw new Error(response.message);
             }
-            fetchChatReportsByIDs([chatReportID]);
 
-            // If an iouReport is open (has an IOU, but is not yet paid) then we sync the chatReport's 'iouReportID'
-            // field in Onyx, simplifying IOU data retrieval and reducing necessary API calls when displaying IOU
-            // components. If we didn't sync the reportIDs, the paid IOU would still be shown to users as unpaid. The
-            // iouReport being fetched here must be open, because only an open iouReoport can be paid.
-            // Therefore, we should also sync the chatReport after fetching the iouReport.
-            fetchIOUReportByIDAndUpdateChatReport(reportID, chatReportID);
+            // Save the updated chat and IOU reports sent back in the response
+            // NOTE: since the API doesn't handle syncing chat reports with IOU reports,
+            // we also need to set the iouReportID and hasOutstandingIOU fields of the chatReport in Onyx manually
```

Comment:
NAB grammar
```suggestion
            // Save the updated chat and IOU reports sent back in the response.
            // NOTE: Since the API doesn't handle syncing chat reports with IOU reports,
            // we also need to set the iouReportID and hasOutstandingIOU fields of the chatReport in Onyx manually.
```